src/bbs2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -256,6 +257,11 @@ private string GetAzureStorageConnectionString(MigrateRepoCommandArgs args) => a
 
     private string GetSmbPassword(MigrateRepoCommandArgs args) => args.SmbPassword.HasValue() ? args.SmbPassword : _environmentVariableProvider.SmbPassword(false);
 
+    private string GetBbsRepoUrl(MigrateRepoCommandArgs args)
+    {
+        return args.BbsServerUrl.HasValue() && args.BbsProject.HasValue() && args.BbsRepo.HasValue() ? $"{args.BbsServerUrl.TrimEnd('/')}/projects/{args.BbsProject}/repos/{args.BbsRepo}/browse" : "https://not-used";
```

Comment:
This is a looong ternary.  Perhaps we can use a guard instead?

```suggestion
        if (!args.BbsServerUrl.HasValue() || !args.BbsProject.HasValue() || !args.BbsRepo.HasValue())
        {
          return "https://not-used";
        }
        
        return $"{args.BbsServerUrl.TrimEnd('/')}/projects/{args.BbsProject}/repos/{args.BbsRepo}/browse"
```

Reply:
Sure.

Reply:
So the linter is complaining about wanting to use ternary expression vs if else 😅 

```
@dpmex4527 ➜ /workspaces/gh-gei (bbs-send-repo-url-for-selecting-fork-support) $ dotnet format src/OctoshiftCLI.sln --verify-no-changes
/workspaces/gh-gei/src/bbs2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs(262,9): warning IDE0046: 'if' statement can be simplified [/workspaces/gh-gei/src/bbs2gh/bbs2gh.csproj]
```

https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0046

Thoughts on how to clear this error?

Reply:
if you run `dotnet format` in your workspace what does it try to change it to?

Reply:
```
        return !args.BbsServerUrl.HasValue() || !args.BbsProject.HasValue() || !args.BbsRepo.HasValue()
            ? "https://not-used"
            : $"{args.BbsServerUrl.TrimEnd('/')}/projects/{args.BbsProject}/repos/{args.BbsRepo}/browse";
```

😅 . At least it's not a one liner. Good with that change?

Reply:
Changes pushed in https://github.com/github/gh-gei/pull/1002/commits/68dd277d42f5e38cff97620fb4aff8f59fde76c8

Reply:
Looks good to me!